clickhouse_driver/connection.py

```diff
@@ -121,7 +125,11 @@ def __init__(
         super(Connection, self).__init__()
 
     def get_description(self):
-        return '{}:{}'.format(self.host, self.port)
```

Comment:
Client is connected to the one particular server at once. This server and port should be in Connection's description.